pyomo/solvers/plugins/solvers/cplex_direct.py

```diff
@@ -160,7 +160,7 @@ def _process_stream(arg):
     def _get_expr_from_pyomo_repn(self, repn, max_degree=2):
         referenced_vars = ComponentSet()
 
-        degree = canonical_degree(repn)
+        degree = repn.polynomial_degree()
         if (degree is None) or (degree > max_degree):
```

Comment:
This method needs updated (similar to gurobi_direct.py). I may take a stab at this too.

Reply:
I'm not sure what you mean.  Can you suggest a change?